internal/server/firm_deputy_hub.go

```diff
@@ -6,15 +6,12 @@ import (
 	"strings"
 )
 
-type FirmHubInformation interface {
```

Comment:
Is there a reason for swapping these specific interfaces for the full `ApiClient`? Interfaces should always be as specific as possible and I don't see what value removing these have over the existing implementation.